sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_task_adherence/task_adherence.prompty

```diff
@@ -25,93 +25,102 @@ inputs:
 
-**Level 2: Barely Adherent**
+**Explanation:** The response is entirely off-topic. It doesn't attempt to answer the user's question or use the available tool.
 
-**Definition:**
-Response partially aligns with instructions but has critical gaps.
+[TaskAdherence: 2] (Barely Adherent)
+**Definition:** Response minimally aligns with the task, showing surface-level relevance but ignoring most instructions. It lacks necessary detail and structure to be helpful.
 
 **Example:**
-  **Query:** What is a recommended weekend itinerary in Paris?
-  **Response:** Spend your weekend visiting famous places in Paris.
-
-Explanation: While the response hints at visiting well-known sites, it is extremely vague and lacks specific details, such as which sites to visit or any order of activities, leaving major gaps in the instructions.
+    **Query:** Recommend a historical fiction book.
+    **Response:** There are many good books in historical fiction.
+    **Tool Definitions:** [{ **name**: "get_book", **description**: "Retrieve a book recommendation for a specified genre.", **parameters**: { "genre": "string" } }]
 
+    **Explanation:** The response references the genre but gives no specific recommendation and ignores the tool, leaving major task gaps.
 
-**Level 3: Moderately Adherent**
-
-**Definition:**
-Response meets the core requirements but lacks precision or clarity.
+[TaskAdherence: 3] (Moderately Adherent)
+**Definition:** Response addresses the main task and includes some relevant content, but key elements are missing or underdeveloped. It may lack completeness, structure, or sufficient clarity to fully satisfy the task.
 
 **Example:**
-  **Query:** What is a recommended weekend itinerary in Paris?
-  **Response:** Visit the Eiffel Tower and the Louvre on Saturday, and stroll through Montmartre on Sunday.
+    **Query:** Recommend a historical fiction book.
+    **Response:** You might like **The Book Thief** by Markus Zusak.
+    **Tool Definitions:** [{ **name**: "get_book", **description**: "Retrieve a book recommendation for a specified genre.", **parameters**: { "genre": "string" } }]
 
-Explanation: This answer meets the basic requirement by naming a few key attractions and assigning them to specific days. However, it lacks additional context, such as timings, additional activities, or details to make the itinerary practical and clear.
+    **Explanation:** The assistant gives a relevant book, showing moderate task adherence. However, it does not use the tool or give supporting information.
 
+## [TaskAdherence: 4] (Mostly Adherent)
 
-**Level 4: Mostly Adherent**
+**Definition:** Response is clear, accurate, and aligns with instructions with minor issues.
 
-**Definition:**
-Response is clear, accurate, and aligns with instructions with minor issues.
+    **Example:**
+    **Query:** Recommend a historical fiction book.
+    **Response:**
+    Let me check...
+    → **[Tool Call: get_book({ "genre": "historical fiction" })]**
```

Comment:
is Tool Call and Tool Result aligned with the response structure schema we have finalized? Please frame the Response accordingly.